notificationworkerlambda/src/main/scala/com/gu/notifications/worker/AndroidSender.scala

```diff
@@ -61,9 +59,13 @@ class AndroidSender(val config: FcmWorkerConfiguration, val firebaseAppName: Opt
 
   def reportBatchSuccesses[C <: DeliveryClient](chunkedTokens: ChunkedTokens, sentTime: Long, functionStartTime: Instant, sqsMessageBatchSize: Int): Pipe[IO, Either[DeliveryException, BatchDeliverySuccess], Unit] = { input =>
     val notificationLog = s"(notification: ${chunkedTokens.notification.id} ${chunkedTokens.range})"
+    val enableAwsMetric = chunkedTokens.notification.dryRun match {
+      case Some(true) => false
+      case _ => true
+    }
     input
       .fold(SendingResults.empty) { case (acc, resp) => SendingResults.aggregateBatch(acc, chunkedTokens.tokens.size, resp) }
-      .evalTap(logInfoWithFields(logFields(env, chunkedTokens.notification, chunkedTokens.tokens.size, sentTime, functionStartTime, Configuration.platform, isIndividualNotificationSend = false, sqsMessageBatchSize), prefix = s"Results $notificationLog: "))
+      .evalTap(logInfoWithFields(logFields(env, chunkedTokens.notification, chunkedTokens.tokens.size, sentTime, functionStartTime, Configuration.platform, sqsMessageBatchSize), prefix = s"Results $notificationLog: ").andThen(_.map(cloudwatch.sendPerformanceMetrics(env.stage, enableAwsMetric))))
```

Comment:
Are we going to change back to embedded log format to send metric to AWS?  Is it part of this PR?

I have this question because you have not added the `_aws` attribute to the logging in `Logging.scala`.

Reply:
I did wonder about this. I reverted back to embedded metrics but then I saw Jacob's recent PR and changed my mind: https://github.com/guardian/mobile-n10n/pull/884#issuecomment-1359197473

At the moment I wondered whether it might be better to favour consistency over any cost-benefit. It also looks like there might be a further department-wide migration of how we send metrics...

Reply:
Ahhh great :) Thanks for raising this!